src/dump/dump.c

```diff
@@ -109,9 +109,6 @@ void dump_write_score(FILE *f, gregorio_score *score)
     if (score->score_copyright) {
         fprintf(f, "   score_copyright           %s\n", score->score_copyright);
     }
-    if (score->language) {
-        fprintf(f, "   language                  %s\n", score->language);
-    }
```

Comment:
Why are we deleting this header field?  Is it present somewhere else in the file?